JavaScripit/index.js

```diff
@@ -0,0 +1,28 @@
+var today = new Date();
```

Comment:
```suggestion
const today = new Date();
```
`const` is preferred, or `let` if the value needs to change.